### ISSUE TITLE
Add color for `tree.indentGuidesStroke`

### ISSUE THREAD

**Issue by octref** [arcticicestudio/nord-visual-studio-code#145]
In the upcoming release, VS Code will add indentation guides in File Explorer. See https://github.com/microsoft/vscode/issues/17777 for details.

A new key `tree.indentGuidesStroke` will be added. It doesn't look nice in Nord yet:

![image](https://user-images.githubusercontent.com/4033249/60267720-b9722b80-989f-11e9-900d-09c194222b76.png)

So I tried 

```json
{
  "workbench.colorCustomizations": {
    "[Nord]": {
      "tree.indentGuidesStroke": "#EBCB8B"
    }
  }
}
```

![image](https://user-images.githubusercontent.com/4033249/60267653-93e52200-989f-11e9-9b51-d790983cf648.png)

Please consider adding a color for `tree.indentGuidesStroke`. Thanks!

**Comment by arcticicestudio** [comment]
Hi @octref :wave:, thanks for your contribution :+1:
Of course the new key will be added, but I guess it makes sense to use the same colors like for the `editorIndentGuide.background` (`nord2` with 70% opacity) and `editorIndentGuide.activeBackground` (`nord3`)  keys.

Also the idea of having a tool to check for missing rules is awesome :100: 
I've commented in the issue you've opened to add some positive arguments for such a nice idea :smile:

**Comment by octref** [comment]
The issue I see with nord3 is that on hover, VS Code will take a dimmed color of the key, and nord3 is a bit too low contrast...

![image](https://user-images.githubusercontent.com/4033249/60316983-d7807000-9921-11e9-93c2-b3a0804e9ed1.png)

I hope a sensible default can be chosen for Nord, but I'll still likely use the color for open folder https://raw.githubusercontent.com/ayu-theme/vscode-ayu/master/icons/folder_opened_dark%402x.png so it's consistent with the icon colors.

![image](https://user-images.githubusercontent.com/4033249/60317157-ab192380-9922-11e9-8b27-a65c0d179366.png)

> Also the idea of having a tool to check for missing rules is awesome 💯
I've commented in the issue you've opened to add some positive arguments for such a nice idea 😄

I didn't see the comment 😅Maybe you forgot to post?

**Comment by arcticicestudio** [comment]
Oh the `nord3` contrast for small areas might be a problem (again 😄), but I guess the new comment color from #117 / #118 could fit. Using the same color like icons might be a problem since almost every (icon) theme comes with different styles and often allows the user to define custom colors, e.g. I use the [Material Icon Theme][mit] with custom Nord colors through the `material-icon-theme.folders.color` setting key.

I started to write the comment in the other ticket, but had to leave for work earlier and saved as draft. Should be visible now :D

[mit]: https://marketplace.visualstudio.com/items?itemName=PKief.material-icon-theme

**Comment by arcticicestudio** [comment]
🚢 Shipped with extension release version 🏷 [0.11.0](https://github.com/arcticicestudio/nord-visual-studio-code/releases/tag/v0.11.0).
